src/main/java/nextstep/MemberData.java

```diff
@@ -0,0 +1,22 @@
+package nextstep;
+
+import lombok.AllArgsConstructor;
+import nextstep.member.domain.Member;
+import nextstep.member.domain.MemberRepository;
+import nextstep.member.domain.RoleType;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+public class MemberData {
+    private static final String ADMIN_EMAIL = "admin@email.com";
+    private static final String ADMIN_PASSWORD = "password";
+    private static final int ADMIN_AGE = 20;
+
+    private static final String MEMBER_EMAIL = "email@email.com";
+    private static final String MEMBER_PASSWORD = "password";
+    private static final int MEMBER_AGE = 20;
+
+    public static Member admin = new Member(ADMIN_EMAIL, ADMIN_PASSWORD, ADMIN_AGE, List.of(RoleType.ROLE_ADMIN.toString()));
```

Comment:
아래와 같이 구현해도 의도대로 동작합니다 
```suggestion
    public static Member admin = new Member(ADMIN_EMAIL, ADMIN_PASSWORD, ADMIN_AGE, List.of(RoleType.ROLE_ADMIN.name()));
```